client/data/authorizations/types.d.ts

```diff
@@ -0,0 +1,41 @@
+/** @format */
```

Comment:
I think we merged this file with the other types file yesterday. We might need to do the same here.

Reply:
Indeed, this must be coming from a bad merge, thanks for catching it! Removed here [28114af](https://github.com/Automattic/woocommerce-payments/pull/5006/commits/28114afe5af3b0097c91eeb84cad3d98c3d9be4c)

Reply:
Thanks!